eng/scripts/Copy-ApiVersion.ps1

```diff
@@ -0,0 +1,77 @@
+[CmdletBinding()]
```

Comment:
I would appreciate some kind of comment explaining why and when I would need/want to use this script.

Reply:
Yes :) @JonathanCrd is going to take over this. When this happens we will make sure to update documentation and all of that.
Thank you @heaths !!!!